src/classes/RD_RecurringDonations_TEST2.cls

```diff
@@ -83,7 +83,7 @@ private with sharing class RD_RecurringDonations_TEST2 {
         string closedstage = [select masterlabel from opportunitystage where isActive = true and iswon = true and isClosed = true limit 1].masterlabel;
         o.StageName = closedstage;
         o.CloseDate = system.today();
-        RD_ProcessControl.hasRun = false;
+        RD_ProcessControl.setRecursionFlag(false);
```

Comment:
if we are now correctly incr/dec our flag in our TDTM code, why is setting the flag needed in our test code?  we've fixed this in all our other components.